test/unit/model-qoe-test.js

```diff
@@ -16,29 +16,29 @@ describe('Model QoE', function() {
         initQoe(model, program);
         const qoeItem = model._qoeItem;
 
-        mediaModel.set('state', STATE_IDLE);
+        mediaModel.set('mediaState', STATE_IDLE);
```

Comment:
Can you add a constant for `mediaState`?

Reply:
Sure, but the constants we have now are really only for events, not for model state properties.